src/app/parent-component/child-component/child-component.component.ts

```diff
@@ -1,17 +1,24 @@
-import { Component, OnInit, Input } from '@angular/core';
+import { Component, OnInit, Input, Output, EventEmitter } from '@angular/core';
 
 @Component({
   selector: 'app-child-component',
-  template: `<h1>{{title}}</h1>`,
+  template: `<h1>{{title}}</h1>
+              <br>
+              <button type="button" (click)="notifyParent($event)">Click Me!</button>`,
```

Comment:
you don't need to pass event here since you are not doing anything with it in notifyParent method